arm/compute/models.go

```diff
@@ -195,6 +221,81 @@ const (
 	// StandardA9 specifies the standard a9 state for virtual machine size
 	// types.
 	StandardA9 VirtualMachineSizeTypes = "Standard_A9"
+	// StandardD1 specifies the standard d1 state for virtual machine size
+	// types.
+	StandardD1 VirtualMachineSizeTypes = "Standard_D1"
+	// StandardD11 specifies the standard d11 state for virtual machine size
+	// types.
+	StandardD11 VirtualMachineSizeTypes = "Standard_D11"
+	// StandardD11V2 specifies the standard d11v2 state for virtual machine
+	// size types.
+	StandardD11V2 VirtualMachineSizeTypes = "Standard_D11_v2"
+	// StandardD12 specifies the standard d12 state for virtual machine size
+	// types.
+	StandardD12 VirtualMachineSizeTypes = "Standard_D12"
+	// StandardD12V2 specifies the standard d12v2 state for virtual machine
+	// size types.
+	StandardD12V2 VirtualMachineSizeTypes = "Standard_D12_v2"
+	// StandardD13 specifies the standard d13 state for virtual machine size
+	// types.
+	StandardD13 VirtualMachineSizeTypes = "Standard_D13"
+	// StandardD13V2 specifies the standard d13v2 state for virtual machine
```

Comment:
It comes directly from the Swagger. Without very special checks, we cannot detect and alter such things. We could raise an issue with the Swagger.